.github/workflows/test.yaml

```diff
@@ -268,6 +268,89 @@ jobs:
             deploy/controller-test-dask-gateway
             deploy/traefik-test-dask-gateway
 
+
+  kubernetes-optional-traefik-test:
+    name: Test k8s backend & chart with traefik installed
+    runs-on: ubuntu-24.04
+    timeout-minutes: 15
+
+    strategy:
+      fail-fast: false
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          # chartpress requires git history to set chart version and image tags
+          # correctly
+          fetch-depth: 0
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.13"
+
+      # Starts a k8s cluster with NetworkPolicy enforcement and installs both
+      # kubectl and helm
+      #
+      # ref: https://github.com/jupyterhub/action-k3s-helm/
+      #
+      - uses: jupyterhub/action-k3s-helm@v4
+        with:
+          k3s-channel: v1.31
+          metrics-enabled: false
+          traefik-enabled: false
+          docker-enabled: true
+
+      - name: Install Python test requirements
+        run: |
+          DASK_GATEWAY_SERVER__NO_PROXY=true pip install -r tests/requirements.txt
+          pip list
+
+      - name: Generate values.schema.json from YAML equivalent
+        run: resources/helm/tools/generate-json-schema.py
+
+      - working-directory: resources/helm
+        env:
+          # DOCKER_BUILDKIT is required for building images with --mount flags,
+          # as used in dask-gateway/Dockerfile.
+          DOCKER_BUILDKIT: "1"
+        run: chartpress
+
+      - name: Install traefik
+        run: |
+          helm repo add traefik https://helm.traefik.io/traefik
+          helm install traefik traefik/traefik \
+              --namespace traefik \
+              --create-namespace \
+              --set ports.web.nodePort=30200
+
+      - name: helm install
+        run: |
+          helm install \
+              test-dask-gateway \
+              resources/helm/dask-gateway \
+              --values=resources/helm/testing/chart-install-values.yaml \
+              --set traefik.installTraefik=false \
+              --set gateway.backend.namespace=default \
+              --wait \
+              --timeout 1m0s
```

Comment:
I figure we should adopt the `--skip-crd` flag here and install the dask-gateway CRDs first in a standalone command, as that would be the procedure I figure we should recommend to avoid having dask-gateway interfere with traefik CRDs in any way.

Reply:
Done in latest commit